src/go/k8s/webhooks/redpanda/console_webhook.go

```diff
@@ -111,6 +114,7 @@ func (v *ConsoleValidator) InjectDecoder(d *admission.Decoder) error {
 type ConsoleDefaulter struct {
 	Client  client.Client
 	decoder *admission.Decoder
+	Logger  logr.Logger
```

Comment:
Why this logger is added?

Reply:
so the logger can be passed up there where the webhook is registered.